pl_bolts/datamodules/cityscapes_datamodule.py

```diff
@@ -153,11 +161,19 @@ def val_dataloader(self):
 
     def test_dataloader(self):
         """
-        Cityscapes test set uses the test split
+        Cityscapes test set
         """
         transforms = self.default_transforms() if self.test_transforms is None else self.test_transforms
-
-        dataset = self.DATASET(self.data_dir, train=False, download=False, transform=transforms, **self.extra_args)
+        target_transforms = self.default_target_transforms(
+        ) if self.target_transforms is None else self.target_transforms
```

Comment:
```suggestion
        target_transforms = self.default_target_transforms() if self.target_transforms is None else self.target_transforms
```
this is bad formatting

Reply:
Applying this suggestion would result in line too long error. In newer version I applied black autoformatting with `pyproject.toml` from parent `pytorch-lightning` repo.

Reply:
ok, pls see updated version